browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -291,7 +291,7 @@ void SpeedreaderTabHelper::SetTheme(Theme theme) {
 Theme SpeedreaderTabHelper::GetTheme() {
   const Theme theme =
       SpeedreaderServiceFactory::GetForProfile(GetProfile())->GetTheme();
-  if (theme == Theme::kDefault) {
+  if (theme == Theme::kNone) {
     switch (dark_mode::GetActiveBraveDarkModeType()) {
       case dark_mode::BraveDarkModeType::BRAVE_DARK_MODE_TYPE_DARK:
```

Comment:
what is `::BRAVE_DARK_MODE_TYPE_DEFAULT` ?

Reply:
// DEFAULT type acts as two ways depends on system theme mode.
  // If system dark mode is not supported, we override it with channel based
  // policy. In this case, user can see dark or light option in settings.
  // Otherwise, it acts like system dark mode mode. It respects system's dark
  // mode. In this case, user can see all three options in theme settings.

GetActiveBraveDarkModeType never returns default.